### DISCUSSION TITLE
Aurora rotary dimmer disable blue led [Solved]

### DISCUSSION THREAD

**Discussion by mjf2708** [Koenkk/zigbee2mqtt#6062] [Devices]
I have an Aurora AOne AU-A1ZB2WDM wall-mounted dimmer, which has a blue 'backlight' LED. Does anyone know how to turn this off? I can't see an exposed endpoint for this (although Aurora say this can be done through their app, but you have to  link to their proprietary hub to do this).

**Comment by proffalken** [comment]
@mjf2708 - this is marked as "solved", but with no details on how - are you able to explain how you did this please as my wife's not keen on the blue lights from behind the switches! :D

**Comment by mjf2708** [comment]
Hi Matthew,

I’m sorry I can’t help you - I no longer use Zigbee, and I can’t remember what I did to disable the blue lights.

Regards,

Mike

**Reply by proffalken** [reply]
No worries, thanks anyway!

**Reply by andydvsn** [reply]
I've bumped into this by accident - I don't know if this is of any use, but if you send an on/off command to end point 3, cluster ID 0x0006 of the device then the blue LED can be switched on and off. On Hubitat (the platform I use to interface with these) it's:

`he cmd 0x${device.deviceNetworkId} 0x03 0x0006 0x00 {}`

`he cmd 0x${device.deviceNetworkId} 0x03 0x0006 0x01 {}`

I don't know what platform you're using, it just so happens I'm adding support for this to [my Hubitat driver](https://github.com/birdslikewires/hubitat/blob/master/aurora/drivers/aurora_dimmer_a1zb2wdm.groovy) for it right now.

**Reply by proffalken** [reply]
Thanks @andydvsn - I've actually just migrated away from Hubitat and back to Home Assistant!

I'm using Zigbee2MQTT, so now I just need to work out how to send that command from the dev console, really appreciate you taking the time to post this!

**Reply by MJWMJW2** [reply]
@proffalken if you figure this out - please let me know :)

**Reply by proffalken** [reply]
@MJWMJW2 - I've been in contact with Aurora, this is what they said (patched together across various emails!)

```
We can advise the Endpoint 1 controls the Dimming aspect of the Dimmer.
Endpoint 3 control the LED Ring around the button.

It would be the on off cluster and send an off command.
```

I still get an error:

```
Error Publish 'set' 'write' to 'Dining main lights' failed: 'Error: Write 0x000d6ffffe7ef527/3 genOnOff({"onOff":1}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Status 'READ_ONLY')'
```

So I'm not too sure what's going on here and I'm going to log a bug

**Comment by proffalken** [comment]
https://github.com/Koenkk/zigbee2mqtt/issues/11381 logged to re-open this
